client/rest-high-level/build.gradle

```diff
@@ -70,6 +70,8 @@ dependencyLicenses {
   }
 }
 
+compileTestJava.options.compilerArgs << "-Xlint:-unchecked"
```

Comment:
Id much prefer a `@SuppressWarnings("unchecked")` in the spots where this is failing rather than a blanket unchecked addition to the compiler args.

Reply:
true, we use surpression per each test case at the moment. Will follow that convention in HLRC and remove that lint flag from PR

Reply:
true, we use suppression per each test case at the moment. Will follow that convention in HLRC and remove that lint flag from PR

Reply:
is it many places? i can live with it in tests a lot easier than i can live with it in non test cc @nik9000

Reply:
the one I introduced are only in tests (due to hamcrest varargs problem) Haven't seen them in prod code. And since HLRC will be included into other peoples projects, having these warning would definitely be bad